tests/Functional/Platform/AlterColumnTest.php

```diff
@@ -25,12 +25,12 @@ public function testColumnPositionRetainedAfterAltering(): void
 
         $sm         = $this->connection->createSchemaManager();
         $comparator = new Comparator();
-        $diff       = $comparator->diffTable($sm->listTableDetails('test_alter'), $table);
+        $diff       = $comparator->diffTable($sm->getTable('test_alter'), $table);
```

Comment:
The migration of the test suite to the new API is justified in this case since the new API is just a stricter version of the old one.